go.mod

```diff
@@ -10,4 +10,5 @@ require (
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.5.2 // indirect
```

Comment:
This addition is unclear.

Reply:
Looks like leftovers after different approaches